dev/com.ibm.websphere.org.eclipse.microprofile.config.1.3/bnd.bnd

```diff
@@ -26,7 +26,7 @@ Export-Package: \
   org.eclipse.microprofile.config.spi;version=1.2
 
```

Comment:
should be
   org.eclipse.microprofile.config;version=1.0.1
   org.eclipse.microprofile.config.inject;version=1.0
   org.eclipse.microprofile.config.spi;version=1.3